src/sys.c

```diff
@@ -587,6 +587,38 @@ typedef DWORD (WINAPI *GAPC)(WORD);
 #endif
 #endif
 
+static int num_threads_bound(void) JL_NOTSAFEPOINT
+{
+#ifdef _OS_DARWIN_
+    return INT_MAX;
+#else
+    int masksize = uv_cpumask_size();
+    uv_thread_t tid = uv_thread_self();
+    char *cpumask = (char *)calloc(masksize, sizeof(char));
+    int err = uv_thread_getaffinity(&tid, cpumask, masksize);
+    if (err) {
+        free(cpumask);
+#ifdef _OS_WINDOWS_
+// On windows, it seems like this can fail due to access right, presumably
+// because `uv_thread_getaffinity` calls `SetThreadAffinityMask`.
+// TODO: Use `GetProcessAffinityMask` directly?
+        if (err == UV_EBADF) {
+            return INT_MAX;
+        }
+#endif
```

Comment:
I saw errors due to this (e.g. https://build.julialang.org/#/builders/65/builds/3285) so I just added a branch to ignore this. I this fine?

Reply:
Ah, I see this is actually because `uv_thread_self` hasn't been implemented on Windows.

Reply:
Cool! Thanks for debugging + fixing this!

ref https://github.com/libuv/libuv/pull/3357

Reply:
We have libuv/libuv#3357 as of #43219